weave-js/src/components/Form/Select.tsx

```diff
@@ -199,6 +199,14 @@ const getStyles = <
         },
       };
     },
+    menu: baseStyles => ({
```

Comment:
(This change also appears in https://github.com/wandb/weave/pull/919)

Reply:
👍